flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -138,17 +140,27 @@ public static String readFile(File file, String charsetName) throws IOException
         return new String(bytes, charsetName);
     }
 
+    public static String readFile(File file, Charset charset) throws IOException {
+        byte[] bytes = readAllBytes(file.toPath());
+        return new String(bytes, charset);
+    }
+
     public static String readFileUtf8(File file) throws IOException {
-        return readFile(file, "UTF-8");
+        return readFile(file, StandardCharsets.UTF_8);
     }
 
     public static void writeFile(File file, String contents, String encoding) throws IOException {
         byte[] bytes = contents.getBytes(encoding);
         Files.write(file.toPath(), bytes, StandardOpenOption.WRITE);
     }
```

Comment:
This method seems to be unused now.

Reply:
I see, I misunderstood the change and was thinking that the usages of new method replaced the old ones in this PR.

Reply:
I've removed the older `String encoding` versions.